.github/workflows/go.yaml

```diff
@@ -0,0 +1,51 @@
+name: GoLang Tests
+
+on:
+  schedule:
+    - cron: 0 */6 * * *
+  pull_request:
+    branches: [main]
+
+jobs:
+  test:
+    runs-on: public-amd64-small
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Go Action
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.22.5'
+      - name: Install dependencies
+        run: |
+          go install github.com/jstemmer/go-junit-report/v2@latest
```

Comment:
I think idiomatic way is to use a go.mod file and install from that

module example.com/mymodule

go 1.14

require (
    example.com/othermodule v1.2.3
    example.com/thismodule v1.2.3
    example.com/thatmodule v1.2.3
)

replace example.com/thatmodule => ../thatmodule
exclude example.com/thismodule v1.3.0

Reply:
`go-junit-report` and `gotestsum ` are CLI commands. This was how the instructions said to install it.